build/visual-studio/slang/slang.vcxproj

```diff
@@ -98,7 +98,7 @@
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
       <TreatWarningAsError>true</TreatWarningAsError>
-      <PreprocessorDefinitions>_DEBUG;SLANG_DYNAMIC_EXPORT;SLANG_WITHOUT_EMBEDDED_STD_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_DEBUG;SLANG_DYNAMIC_EXPORT;SLANG_WITHOUT_EMBEDDED_STD_LIB;SLANG_ENABLE_IR_BREAK_ALLOC=1;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Minor naming question: is it better to have `SLANG_IR_ENABLE_*` or `SLANG_ENABLE_IR_*`? The latter option (which you currently have) reads more naturally, but means that any flags related to a sub-module of the compiler are not naturally grouped by a prefix. The former option means that all config options/defines related to the IR would start with `SLANG_IR_*` which could be nice.

I have no strong opinion on this right now given that we only have the one option being introduced here.